README.md

```diff
@@ -1,8 +1,8 @@
-# Game Chain Contracts
+# World Chain Contracts
 
-*Disclaimer: This is very much a work in progess.*
+![Project Awakening cover logo](./ProjectAwakeningCover.png)
 
-This repository contains the smart contract code supporting Project Awakening and targeted for deployment on the game chain. It contains of two separate code paths. The standard contracts are plain solidity contracts intended as support structure. The mud contracts are the functionality or *business logic* itself. 
+This repository contains the smart contract code supporting Project Awakening and targeted for deployment on the world chain. It contains of two separate code paths. The standard contracts are plain solidity contracts intended as support structure. The mud contracts are the functionality or *business logic* itself. 
```

Comment:
does world chain the right word  ?

Reply:
Its "a" word. At least for the time being

Reply:
can we rename to `to the world` or `on the game chain`.

Reply:
Sure, renamed in aa51e59